.github/workflows/ci.yml

```diff
@@ -98,3 +98,51 @@ jobs:
           - { EXPECTED_FAILURES: "or1k-cy or1k-cmov", EXTRA_CORE_ARGS: "--feature_cmov NONE" }
           - { EXPECTED_FAILURES: "or1k-cy or1k-ext", EXTRA_CORE_ARGS: "--feature_ext NONE" }
 
+  formal_tests:
```

Comment:
can you change this name to `formal` or `formal-verification`, using `_` underscore is not consistent with other tests